gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -286,6 +288,20 @@ class BaselineIdea extends AbstractBaselinePlugin {
         }
     }
 
+    private static void ignoreCommonShadedPackages(Node node) {
+        // language=xml
+        node.append(new XmlParser().parseText('''
+            <component name="JavaProjectCodeInsightSettings">
+              <excluded-names>
+                <name>shadow</name><!-- from gradle-shadow-jar -->
+                <name>org.gradle.internal.impldep</name>
+                <name>autovalue.shaded</name>
+                <name>org.inferred.freebuilder.shaded</name>
+              </excluded-names>
```

Comment:
@ilyanep If memory serves you had a list of packages that are helpful to exclude, is there anything missing here?

Reply:
@carterkozak My full list is:

1. com.palantir.shaded
2. com.sun.javafx.collections
3. com.sun.tools.javac.util
4. com.sun.xml.internal
5. de.intarsys
6. io.vavr
7. jersey.repackaged
8. org.apache.poi.ss.formula
9. org.aspectj.lang.annotation
10. org.codehaus.jackson
11. org.jooq.Exception.IOException
12. shaded.com.palantir
13. shaded.es244
14. shaded.es6x

Copied from what I have configured right now, though not sure how many of those are generally applicable for baseline. Hope that's helpful!